go/libraries/doltcore/sqle/dfunctions/dolt_merge.go

```diff
@@ -37,6 +37,8 @@ type DoltMergeFunc struct {
 	expression.NaryExpression
```

Comment:
the return type of this function is still `sql.Text`. line 304:
```
func (d DoltMergeFunc) Type() sql.Type {
	return sql.Text
}
```

Reply:
Good catch